pkgs/applications/graphics/unigine-superposition/default.nix

```diff
@@ -0,0 +1,163 @@
+{ lib,
+  glib,
+  stdenv,
+  dbus,
+  freetype,
+  fontconfig,
+  zlib,
+  qtquickcontrols2,
+  libXinerama,
+  libxcb,
+  libSM,
+  libXi,
+  libglvnd,
+  libXext,
+  libXrandr,
+  mailspring,
+  libX11,
+  libICE,
+  libXrender,
+  autoPatchelfHook,
+  makeWrapper,
+  mkDerivation,
+  xkeyboard_config,
+  fetchurl,
+  buildFHSUserEnv,
+  openal
+}:
+
+let
+
+superposition = stdenv.mkDerivation rec{
+  pname = "unigine-superposition";
+  version = "1.1";
+
+  src = fetchurl {
+    url = "https://assets.unigine.com/d/Unigine_Superposition-${version}.run";
+    sha256 = "12hzlz792pf8pvxf13fww3qhahqzwzkxq9q3mq20hbhvaphbg7nd";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    makeWrapper
+  ];
+
+  buildInputs = [
+    glib
+    stdenv.cc.cc
+    dbus
+    freetype
+    fontconfig
+    zlib
+    qtquickcontrols2
+    libXinerama
+    libxcb
+    libSM
+    libXi
+    libglvnd
+    libXext
+    libXrandr
+    mailspring
+    libX11
+    libICE
+    libXrender
+  ];
+
+  installPhase = ''
+    sh $src --target $name --noexec
+
+    mkdir -p $out/lib/unigine/superposition/
+    mkdir $out/bin
+
+    cp -r $name/* $out/lib/unigine/superposition/
+
+    echo "exec $out/lib/unigine/superposition/Superposition" >> $out/bin/superposition
+    chmod +x $out/bin/superposition
+
+    wrapProgram $out/lib/unigine/superposition/Superposition --set QT_XKB_CONFIG_ROOT ${xkeyboard_config} --run "cd $out/lib/unigine/superposition/"
```

Comment:
```suggestion
    wrapProgram $out/lib/unigine/superposition/Superposition \
      --set QT_XKB_CONFIG_ROOT ${xkeyboard_config} \
      --chdir $out/lib/unigine/superposition/
```

Reply:
Builder called die: makeWrapper doesn't understand the arg --chdir

Reply:
Than that is still in staging.